packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -410,6 +436,27 @@ export class GarbageCollector implements IGarbageCollector {
         { end: true, cancel: "error" });
     }
 
+    /**
+     * Adds the most recent GC state to the summary in a tree.
```

Comment:
Since this replaces the old summary GC blob if there is one, would it be reasonable to say updates the summary GC state or adds...?

Reply:
This does not replace old blobs. This is called when a new summary is generated and has to be uploaded to the server.